d2l-tspan.js

```diff
@@ -129,7 +126,7 @@ Polymer({
 	_selectedChanged: function(selected) {
 		this.setAttribute('aria-selected', selected.toString());
 	},
-	_onSizeChanged: function() {
+	_onSlotSizeChanged: function() {
 		if (this.root.host) {
 			this.root.host.style.height = this.$.float.offsetHeight + 'px';
```

Comment:
Could the dimensions be passed to this method, or does the caller not have the dimension that's needed here?